tskv/src/wal/mod.rs

```diff
@@ -129,6 +132,7 @@ impl Display for WalType {
             WalType::DeleteTable => write!(f, "delete_table"),
             WalType::UpdateSeriesKeys => write!(f, "update_series_keys"),
             WalType::Delete => write!(f, "delete"),
+            WalType::ClearWalEntry => write!(f, "clear_wal_entry"),
```

Comment:
To **clear wal entry** we didn't write anything to a WAL file, what does WalType::ClearWalEntry do?

Reply:
not write,just clear files.